micro_pumas_v1.F90

```diff
@@ -1173,7 +1204,12 @@ subroutine micro_pumas_tend ( &
   !$acc               areflz,frefl,csrfl,acsrfl,fcsrfl,rercld,ncai,ncal,      &
   !$acc               qrout2,qsout2,nrout2,nsout2,drout2,dsout2,freqs,        &
   !$acc               freqr,nfice,qcrat,qgout,dgout,ngout,qgout2,ngout2,      &
-  !$acc               dgout2,freqg,prer_evap)                                 &
+  !$acc               dgout2,freqg,prer_evap)
+  !$acc      copyout (nnuccctot,nnuccttot,nnuccdtot,nnudeptot,nhomotot,       &
+  !$acc               nnuccrtot,nnuccritot,nsacwitot,npratot,npsacwstot,      &
+  !$acc               npraitot,npracstot,nprctot,nprcitot,ncsedten,nisedten,  &
+  !$acc               nrsedten,nssedten,ngsedten,nmelttot,nmeltstot,          &
+  !$acc               nmeltgtot)  
```

Comment:
Do we know why these are in their own copyout statement? I think they would be less likely to be lost, forgotten, or cause merge conflicts, if they are included with the other output arguments.